spec/rex/socket_spec.rb

```diff
@@ -150,19 +130,6 @@
         expect(subject).to include("fe80::2")
       end
     end
-
-    context "with rubinius' bug returning ASCII addresses" do
-      let(:response_afamily) { Socket::AF_INET }
-      let(:response_addresses) { ["1.1.1.1", "2.2.2.2"] }
-
-      it { is_expected.to be_an(Array) }
-      it { expect(subject.size).to eq(2) }
-      it "should return the ASCII addresses" do
-        expect(subject).to include("1.1.1.1")
-        expect(subject).to include("2.2.2.2")
-      end
-
-    end
```

Comment:
Is this rubinus bug fixed now?

Reply:
Going by these comments below, the Rubinius bug has been removed due to it been tied to the deprecated method (`::Socket.gethostbyname`). 
```Ruby
# Rubinius has a bug where gethostbyname returns dotted quads instead of
# NBO, but that's what we want anyway, so just short-circuit here.
```

Reply:
Yea, I have removed that logic now 👍